jena-core/src/main/java/org/apache/jena/mem/HashedBunchMap.java

```diff
@@ -24,7 +24,10 @@
 
 /**
     An implementation of BunchMap that does open-addressed hashing.
+    @deperecated
```

Comment:
`@deprecated`

Reply:
Sorry, once one sees it, it really hurts. I fixed it.